settings/__init__.py

```diff
@@ -1,20 +1,16 @@
-import os
 import sys
 
-if os.getenv('TRAVIS', False):
-    from settings.travis import *  # noqa
-else:
+try:
+    from settings.local import *  # noqa
+except ImportError:
     try:
-        from settings.local import *  # noqa
+        from settings.base import *  # noqa
     except ImportError:
-        try:
-            from settings.base import *  # noqa
-        except ImportError:
-            sys.stderr.write(
-                "Error: Tried importing 'settings.local' and 'settings.base' "
-                "but neither could be found (or they're throwing an "
-                "ImportError). Please fix and try again.")
-            raise
+        sys.stderr.write(
+            "Error: Tried importing 'settings.local' and 'settings.base' "
+            "but neither could be found (or they're throwing an "
+            "ImportError). Please fix and try again.")
+        raise
```

Comment:
Since this branch is intended for new deployments only, and there is a relatively small pool of existing development environments, you might consider just dropping settings/local.py entirely and converting settings to just a single file now sooner rather than later-- no need to block this PR on it though, as that change probably belongs in a separate commit anyway.